MonMaraicher/Features/FarmerDetail/FarmerDetailsViewModel.swift

```diff
@@ -36,6 +41,11 @@ struct FarmerDetailsViewModel: Identifiable, Hashable {
         self.city = marker.address.city.capitalized
     }
 
+    func onMailButtonTapped() {
+        guard let email, let url = URL(string: "mailto:\(email)") else { return }
+        UIApplication.shared.open(url)
```

Comment:
You can use the SwiftUI view modifier instead if you want: `openURL`

Reply:
https://developer.apple.com/documentation/swiftui/environmentvalues/openurl

Reply:
It's up to you ;)

Reply:
Yes, in TODO list ;)